custom_otello_linter/visitors/steps_checkers/make_screenshot_checker.py

```diff
@@ -21,41 +26,17 @@ def check_steps(self, context: Context, config) -> List[Error]:
                     or step.name.startswith('and')
                     or step.name.startswith('but')
             ):
-                screenshot_calls = 0
-
-                # Проходим через каждый элемент в теле функции (step.body)
-                for stmt in step.body:
-                    # Применяем ast.walk к каждому выражению в теле функции
-                    for node in ast.walk(stmt):
-                        if isinstance(node, ast.Call):
-                            func_name = self._get_full_func_name(node.func)
-
-                            # Проверяем вызов функции make_screenshot_for_comparison
-                            if func_name.endswith('make_screenshot_for_comparison'):
-                                screenshot_calls += 1
-
-                                # Если вызовов функции больше одного, добавляем ошибку
-                                if screenshot_calls > 1:
-                                    errors.append(MultipleScreenshotsError(
-                                        lineno=node.lineno,
-                                        col_offset=node.col_offset,
-                                        step_name=step.name
-                                    ))
-                                    # Прерываем проверку текущего шага после обнаружения ошибки
-                                    break
+                screenshot_calls = find_make_screenshot_calls(step.body)
+
+                # Если вызовов функции больше одного, добавляем ошибку
+                if len(screenshot_calls) > 1:
+                    errors.append(MultipleScreenshotsError(
+                        lineno=screenshot_calls[1].lineno,
+                        col_offset=screenshot_calls[1].col_offset,
+                        step_name=step.name))
+                    # Прерываем проверку текущего шага после обнаружения ошибки
+                    break
```

Comment:
```suggestion
                    for screenshot_call in screenshot_calls:
                        errors.append(MultipleScreenshotsError(
                            lineno=screenshot_call.lineno,
                            col_offset=screenshot_calls.col_offset,
                            step_name=step.name))
```
1) Не вижу причин не добавить все ошибки, когда есть возможность)
2) Не очень понял, зачем тут break

Reply:
Тут скорее так: 

`                if len(screenshot_calls) > 1:
                    for screenshot_call in screenshot_calls[1:]:  # Начинаем со второго вызова, тк первый не должен вызывать ошибку
                        errors.append(MultipleScreenshotsError(
                            lineno=screenshot_call.lineno,
                            col_offset=screenshot_call.col_offset,
                            step_name=step.name))`
                            
Насчёт break: не нужен, думаю, больше же всё равно ничего не выполняется